CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 ##########################
 ## Set Project version
 ##########################
-cmake_minimum_required(VERSION 3.17)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
Great! Would you please update the min version at `pennylane_lightning_gpu/src/tests/CMakeLists.txt` too?

Reply:
Done